README.md

```diff
@@ -111,12 +112,31 @@ location @extensionless-php {
 }
 ```
 
+If you've put your nginx configuration in a file in `sites_available`, link it and test nginx
+
+```sh
+function link_nginx(){
```

Comment:
not sure if using a function here is really necessary, probably easier for the sake of the tutorial if you just say to `ln -s /etc/nginx/sites-available/$domain /etc/nginx/sites-enabled/$domain`